webknossos/Changelog.md

```diff
@@ -13,6 +13,7 @@ For upgrade instructions, please check the respective _Breaking Changes_ section
 [Commits](https://github.com/scalableminds/webknossos-libs/compare/v0.14.22...HEAD)
 
 ### Breaking Changes
+- Preferring a custom tiff reader over the default PIMS reader to convert tiff files. This enables the recognition of axis information and the support of tifffiles with more than 3 dimensions. Nonetheless this leads to changed behavior when converting tiff files. Tiffs with axes other than c, x, y and z are no longer supported for conversion to WKW, please convert these to Zarr3 Datasets. [1043](https://github.com/scalableminds/webknossos-libs/pull/1043)
```

Comment:
Does converting to WKW still work if there is an additional axis (e.g. t) but with just 1 slice?

Reply:
Currently, it is not supported. I can add a logic for that.

Reply:
I implemented a logic that sets the bounding box to 3d if additional axes have a shape of 1. I couldn't find fitting test data for that case so far.